### PR TITLE
Test ensure_all_started

### PR DESCRIPTION
**Pull request by BunnyDunker**

### CODE REVIEW

**Review by grossvogel — CHANGES_REQUESTED** [review]
Sorry, we actually want to leave SSL on for Heroku https://devcenter.heroku.com/changelog-items/2035

File: lib/slax/release.ex [hunk 1]
```diff
@@ -23,7 +23,7 @@ defmodule Slax.Release do
   end
 
   defp load_app do
-    Application.ensure_started(:ssl)
+    Application.ensure_all_started(:ssl)
```
Comment (pjdiiori) [inline lib/slax/release.ex:26]: wow was this little change the cause of the ssl issue?

**Review by pjdiiori — APPROVED** [review]

File: lib/slax/release.ex (hunk 1 above)
Comment (BunnyDunker) [reply lib/slax/release.ex:26]: We'll find out in a few minutes 😅

Comment (BunnyDunker) [reply lib/slax/release.ex:26]: And actually part of the issue that Jason and I found is that the DB being used is the same as Slack-Agile so it was having trouble with the new migration I created

**Pull request merged**